JavaFiddle-Web/src/main/java/ru/javafiddle/web/models/UserJF.java

```diff
@@ -13,14 +13,18 @@
     private String lastName;
     private String nickName;
     private String email;
+    private String registrationDate;
```

Comment:
Becouse bean returns it in the type of String, in database it also contains as a String(I mean varchar)